app/helpers/question_attribute_helper.rb

```diff
@@ -10,6 +10,7 @@ def stimulus_html_attributes(question)
 private
 
   def numeric_question_html_attributes(question)
+    return { "style": "background-color: #f3f2f1;" } if question.read_only?
```

Comment:
Might be better to add a CSS class?

Reply:
I thought that, but it's just one thing. What are the benefits?

Reply:
Not huge so I think either way is ok. Benefits are consistent place you know to look for styling and colours (it's probably where I'd check first), and quicker to reuse if we did have a second thing at some point?).

Reply:
I can't see us using it somewhere else right now. I'll just merge it in as is